PeripheryFunctions/BF_ClusterDown.m

```diff
@@ -130,7 +130,8 @@
 
 % Get the dendrogram reordering:
 ax1 = subplot(1,6,6);
-[h_dend,~,ord] = dendrogram(links,0,'Orientation','right');
+ord = optimalleaforder(links,squareform(distMat));
```

Comment:
This can be very slow for large matrices, I'm adding a general function that works through these cases, cf. BF_ClusterReorder